src/StudyApi/Controllers/V1/Local/LocalController.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Amazon.CognitoIdentityProvider;
+using Application.Contracts;
+using Application.Responses.V1.Users;
+using Domain.Entities.Participants;
+using Dte.Common.Responses;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace StudyApi.Controllers.V1.Local;
+
+[ApiController]
+[ApiVersion("1")]
+[Route("api/local")]
+public class LocalController : Controller
+{
+    private readonly IParticipantService _participantService;
+    private readonly IParticipantRepository _participantRepository;
+    private readonly IUserService _userService;
+    private readonly ILogger<LocalController> _logger;
+
+    public LocalController(IParticipantService participantService, IParticipantRepository participantRepository,
+        IUserService userService, ILogger<LocalController> logger)
+    {
+        _participantService = participantService;
+        _participantRepository = participantRepository;
+        _userService = userService;
+        _logger = logger;
+    }
+
+    /// <summary>
+    /// [AllowAnonymous] Login
+    /// </summary>
+    /// <response code="200">When IsSuccess true</response>
+    /// <response code="500">Server side error</response>
+    [AllowAnonymous]
+    [SwaggerResponse(StatusCodes.Status200OK, Type = typeof(Response<UserLoginResponse>))]
+    [SwaggerResponse(StatusCodes.Status500InternalServerError, Type = null)]
+    [HttpPost("UpdateLinkedRecords")]
+    public async Task<IActionResult> UpdateLinkedRecords()
```

Comment:
I know this is a local harness but consider capturing the request `CancellationToken` here and passing it down through the layers.